cliquet/listeners/redis.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import absolute_import
+from six.moves.urllib import parse as urlparse
+import redis
+import json
+
+from cliquet.listeners import ListenerBase
+
+
+class RedisListener(ListenerBase):
+    def __init__(self, *args, **kwargs):
+        super(RedisListener, self).__init__(*args, **kwargs)
+        maxconn = kwargs.pop('max_connections')
+        self._listname = kwargs.pop('listname', 'cliquet.events')
+        connection_pool = redis.BlockingConnectionPool(max_connections=maxconn)
+        self._client = redis.StrictRedis(connection_pool=connection_pool,
+                                         **kwargs)
+
+    def __call__(self, event):
+        try:
+            payload = json.dumps(event.payload)
+        except Exception:
+            # XXX we should log this
+            return
+
+        try:
+            self._client.lpush(self._listname, payload)
+        except Exception:
+            # XXX we should log this
```

Comment:
Here, we should probably use [the ``wrap_redis_error`` decorator](https://github.com/mozilla-services/cliquet/blob/master/cliquet/storage/redis.py#L14)